ibm/service/backuprecovery/resource_ibm_backup_recovery_connection_registration_token.go

```diff
@@ -0,0 +1,138 @@
+// Copyright IBM Corp. 2024 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+/*
+ * IBM OpenAPI Terraform Generator Version: 3.94.0-fa797aec-20240814-142622
+ */
+
+package backuprecovery
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/conns"
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/IBM/go-sdk-core/v5/core"
+	"github.com/IBM/ibm-backup-recovery-sdk-go/backuprecoveryv1"
+)
+
+func ResourceIbmBackupRecoveryConnectionRegistrationToken() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceIbmBackupRecoveryConnectionRegistrationTokenCreate,
+		ReadContext:   resourceIbmBackupRecoveryConnectionRegistrationTokenRead,
+		DeleteContext: resourceIbmBackupRecoveryConnectionRegistrationTokenDelete,
+		UpdateContext: resourceIbmBackupRecoveryConnectionRegistrationTokenUpdate,
+		Importer:      &schema.ResourceImporter{},
+		CustomizeDiff: checkDiffResourceIbmBackupRecoveryConnectionRegistrationToken,
+		Schema: map[string]*schema.Schema{
+			"connection_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				// ForceNew:    true,
+				Description: "Specifies the ID of the connection, connectors belonging to which are to be fetched.",
+			},
+			"x_ibm_tenant_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Required: true,
+				// ForceNew:    true,
+				Description: "Specifies the key to be used to encrypt the source credential. If includeSourceCredentials is set to true this key must be specified.",
+			},
+			"registration_token": &schema.Schema{
```

Comment:
If this is secret value can we add sensitive : true so it mask the value in logs/output

Reply:
@hkantare a connection registration_token is something the user would like to use for registering connectors on this connection. So we would require this in output.